docs/book/src/cronjob-tutorial/testdata/emptymain.go

```diff
@@ -89,6 +90,40 @@ func main() {
 		os.Exit(1)
 	}
 
+	/*
+
+	Note that the Manager can restrict the namespace that all controllers will watch for resources by:
+
+	```
```

Comment:
Use ```go

Reply:
I noticed that it doesn't look great in netlify preview, use some whitespaces should do the trick.
```
	```go
	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
	    Scheme: scheme,
	    Namespace: namespace,
	    MetricsBindAddress: metricsAddr,
	})
	```
```

Reply:
How could I check it in netlify preview? Could you pass the link?

Reply:
Tried to fix the spaces anyway.

Reply:
It's in the github check list.
Here's the link: https://deploy-preview-1229--kubebuilder.netlify.com/

Reply:
Really tks I did not found that.

Reply:
It is definitely something that we could add in the CONTRIBUTION.MD